baseplate/__init__.py

```diff
@@ -93,8 +96,90 @@ def make_tracing_client(raw_config, log_if_unconfigured=True):
     )
 
 
+def crash_reporter_from_config(raw_config, module_name):
+    """Configure and return a crash reporter.
+
+    This expects one configuration option and can take many optional ones:
+
+    ``sentry.dsn``
+        The DSN provided by Sentry.
+    ``sentry.site`` (optional)
+        An arbitrary string to identify this client installation.
+    ``sentry.environment`` (optional)
+        The environment your application is running in.
+    ``sentry.exclude_paths`` (optional)
+        Comma-delimited list of module prefixes to ignore when discovering
+        where an error came from.
+    ``sentry.include_paths`` (optional)
+        Comma-delimited list of paths to include for consideration when
+        drilling down to an exception.
+    ``sentry.ignore_exceptions`` (optional)
+        Comma-delimited list of fully qualified names of exception classes
+        (potentially with * globs) to not report.
+    ``sentry.sample_rate`` (optional)
+        Percentage of crashes to report. (e.g. 37%)
```

Comment:
Yeah, scrolling down showed that, but I don't think it'd be super obvious just by reading the docstring.